### ISSUE TITLE
How to replace text already present in a text field whose accessibility label is know

### ISSUE THREAD

**Issue by Sean-Wang** [kif-framework/KIF#163]
I'm using this tool for automation of my app. I have tried to use this:
    `[scenario addStep:[KIFTestStep stepToEnterText:@"user@example.com" intoViewWithAccessibilityLabel:@"電子郵箱"]];
    [scenario addStep:[KIFTestStep stepToEnterText:@"thisismypassword" intoViewWithAccessibilityLabel:@"請輸入密碼"]];
    [scenario addStep:[KIFTestStep stepToTapViewWithAccessibilityLabel:@"登入"]];`

But it just append "user@example.com" into the string already present in the text field also show the test as failed:

`2012-10-25 15:46:44.082 sti1 copy[23441:c07] FAIL (1.87s): Type the text "user@example.com" into the view with accessibility label "電子郵箱"
2012-10-25 15:46:44.083 sti1 copy[23441:c07] FAILING ERROR: Error Domain=KIFTest Code=0 "Failed to actually enter text "user@example.com" in field; instead, it was "電子郵箱"" UserInfo=0x89dab00 {NSLocalizedDescription=Failed to actually enter text "user@example.com" in field; instead, it was "電子郵箱"}`

Can you please guide me to achieve this?
